vm/systemSmartContracts/eei.go

```diff
@@ -272,6 +280,63 @@ func (host *vmContext) Transfer(
 	return nil
 }
 
+// ProcessBuiltInFunction will execute process if sender and destination is same shard/sovereign
```

Comment:
[comment]
Although this is being tested in other parts, we could've used dedicated test for this